src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const discountOfSevenDays = 50;
+  const discountOfThreeOrMoreDays = 20;
+  const total = days * 40;
+
+  if (days >= 7) {
+    return total - discountOfSevenDays;
+  }
 
+  if (days >= 3 || days >= 6) {
+    return total - discountOfThreeOrMoreDays;
+  }
+
+  return 80;
+}
```

Comment:
The function `calculateRentalCost` is well named, clearly indicating its purpose. However, there are some issues with the implementation:
- The value `40` used in the calculation of `total` should be extracted into a constant to avoid hardcoding and make the code more maintainable.
- The condition `days >= 3 || days >= 6` in line 15 is redundant because `days >= 6` is covered by the first part of the condition `days >= 3`. This should be corrected to just `days >= 3`.
- The return value `80` in line 19 is a hardcoded value that does not match the expected logic. It should be `total` instead, as the calculation should return the total cost without any discounts if none of the conditions for a discount are met.